pcdsdevices/digital_signals.py

```diff
@@ -0,0 +1,20 @@
+from ophyd import Component as Cpt
+from ophyd import Device
+
+from .interface import BaseInterface
+from .signal import PytmcSignal
+
+
+class J120K(BaseInterface, Device):
+    """
+    A class representing the J120K 24V dry contact cooling flow switch.
+    """
+    flow_ok = Cpt(PytmcSignal, ':FSW:FLOW_OK', io='i',
+                  kind='normal', doc='flow rate nominal')
+
+    @property
+    def get_flow_ok(self):
```

Comment:
A few more thoughts, though I'm totally splitting hairs at this point:

- Why does this property exist? Is `flow_ok.get()` insufficient as an API? (I do see that it returns an int and not a bool)
- If there's a reason for it to exist, why is it a property and not a method?
- If there's a good reason for it to be a property, why is it named like a function? I think people in an interactive session will very likely do a `get_flow_ok()` call which will give an annoying "type `bool` is not callable" sort of error.

Reply:
This is actually totally true, I did this myself. I'll delete it. I thought people would want a designated function, but in this case it should drop the property tag. I also did think about the fact that you can just call get on the signal.

Reply:
ok, sounds good- I would have been fine with any outcome here, just trying to make sure we consider these small details